scripts/bash/enceladus_env.template.sh

```diff
@@ -34,12 +34,14 @@ STD_DEFAULT_DRIVER_MEMORY=""
 STD_DEFAULT_DRIVER_CORES=""
 STD_DEFAULT_EXECUTOR_MEMORY=""
 STD_DEFAULT_EXECUTOR_CORES=""
+STD_DEFAULT_DRA_EXECUTOR_MEMORY=""
```

Comment:
How about this is set based on the executor-memory (if passed in)?

Reply:
So I intentionally have them separated the normal executor memory and the DRA executor memory, since when you sent executor memory during the DRA it behaves like the "initial number of executor" and will never scale below this number. Even if there is no reason to have X executors, it will keep them.

Reply:
--num-executors is the 'initial number of executor' in DRA. executor-memory is pretty much the same w and w/o DRA =memory per executor.

Reply:
Also, if we override some of the params for DRA, I would recommend to override all of them together. Otherwise, if someone wants '4 executors x 8 cores + 24 GB' and we override only part of it, it may result with something like 'DRA with 1 core and 24 GB per executor'.